### PR TITLE
Add a config option to disallow links for non-ac users

### PR DESCRIPTION
**Pull request by bumbadadabum**

### CODE REVIEW

File: chat.js [hunk 1]
```diff
@@ -745,6 +745,14 @@ class CommandContext {
 				return false;
 			}
 
+			// If the corresponding config option is set, non-AC users cannot send links, except to staff.
+			if (Config.restrictLinks && !user.autoconfirmed && message.match(linkRegex)) {
+				if (!(targetUser && targetUser.can('lock'))) {
+					this.errorReply("Your account must be autoconfirmed to send links to other users, except for staff.");
```
Comment (panpawn) [inline chat.js]: I think this should say "except for global staff"

**Comment by Zarel** [comment]
Do we all support this, though?

**Comment by AustinXII** [comment]
I think so, yes

**Comment by bumbadadabum** [comment]
@Zarel the SFC thread seems entirely positive and my post about the PR got quite a few likes, as well as the policy admin supporting it. I think we support this enough.

File: chat.js (hunk 1 above)
Comment (bumbadadabum) [reply chat.js]: good point

**Pull request merged**